README.md

```diff
@@ -319,15 +312,19 @@ being used to sign packages, in order to prevent cross-protocol attacks.
 
 #### Validating resources
 
-For a resource to be valid, its hash needs to match all of the hashes provided
-in the manifest, although, like
+For a resource to be valid, its hash needs to appear in the list for each
```

Comment:
The idea is that we make a set of the allowed `hash-value`s, hash each resource, and then the resource is allowed if its hash appears in the set. I'll try to describe that more clearly.